lib/tests/post_iris_call_receive_header.js

```diff
@@ -0,0 +1,63 @@
+const expect = require('chai').expect;
+const { getRskTransactionHelper } = require('../rsk-tx-helper-provider');
+const { getBtcClient } = require('../btc-client-provider');
+const { getBridge, getLatestActiveForkName } = require('../precompiled-abi-forks-util');
+const { activateFork, sendTxWithCheck, getLatestForkName, waitAndUpdateBridge } = require('../rsk-utils');
+const { ensure0x } = require('../utils');
+
+let rskTxHelper;
+let btcTxHelper;
+let bridge;
+
+const fulfillRequirementsToRunAsSingleTestFile = async () => {
+    const forkName = process.env.FORK_NAME || getLatestForkName().name;
+    await activateFork(Runners.common.forks[forkName]);
+};
+
+const execute = (description, getRskHost) => {
```

Comment:
Not part of this ticket but I guess we should consider adding new test cases for the different error codes this method can return. 

cc @ed-iov